examples/trials/cifar10_pytorch/search_space.json

```diff
@@ -1,5 +1,5 @@
-{
-    "lr":{"_type":"choice", "_value":[0.1, 0.01, 0.001, 0.0001]},
-    "optimizer":{"_type":"choice", "_value":["SGD", "Adadelta", "Adagrad", "Adam", "Adamax"]},
-    "model":{"_type":"choice", "_value":["vgg", "resnet18", "googlenet", "densenet121", "mobilenet", "dpn92", "senet18"]}
+{
+    "lr":{"_type":"choice", "_value":[0.1, 0.01, 0.001, 0.0001]},
+    "optimizer":{"_type":"choice", "_value":["SGD", "Adadelta", "Adagrad", "Adam", "Adamax"]},
+    "model":{"_type":"choice", "_value":["vgg", "resnet18", "googlenet", "densenet121", "mobilenet", "dpn92", "senet18"]}
```

Comment:
why update this?

Reply:
Sorry, I have not noticed that before because this two file looks the same.